eslint/babel-eslint-parser/src/worker/configuration.cts

```diff
@@ -73,22 +73,34 @@ function validateResolvedConfig(
         throw new Error(error);
       }
     }
-    if (config.options) return config.options;
   }
 
-  return getDefaultParserOptions(parseOptions);
-}
+  if (process.env.BABEL_8_BREAKING) {
```

Comment:
Why can't `@babel/eslint-parser` release a major without bumping other packages?

Reply:
Maybe it's just for simplicity. :)
But it may change in the future.